File: to_img.php
<?php
$dir = "./out/";
$scanned_directory = glob('./out/*.{svg}', GLOB_BRACE);

foreach ($scanned_directory as $value) {
    $outputf = str_replace("out", "img", $value);
    $outputf = str_replace(".svg", ".png", $outputf);
    $cmd = "rsvg-convert " . $value . " -o " . $outputf . " -w 30 -a";
    `$cmd`;
}

File: to_img.py
import glob
import os

from pprint import pprint
files_in = glob.glob('./out/*.svg')
for x in files_in:
    print(x)
    #os.system("rsvg-convert " + x + " -o " + x.replace("out","img").replace("svg", "png") + " -w 30 -a")
